util/printchplenv

```diff
@@ -132,6 +128,32 @@ def print_mode(mode='list'):
         print_mode('runtime')
         stdout.write('CHPL_MAKE_LAUNCHER_SUBDIR=')
         print_mode('launcher')
+
+        third_party_pkgs = chpl_home+'/util/chplenv/third-party-pkgs'
```

Comment:
I thought about this also, and even took a step down that path, but it quickly looked too hard for my limited skillz.